Panic with uptime collector

```
panic: runtime error: index out of range

goroutine 94366 [running]:
github.com/virtua-network/smartos_exporter/collector.(*LoadAverageExporter).parseUptimeOutput(0xc420158300, 0xc420014370, 0x4c, 0x600, 0xc420014370)
        /root/work/src/github.com/virtua-network/smartos_exporter/collector/uptime.go:70 +0x243
github.com/virtua-network/smartos_exporter/collector.(*LoadAverageExporter).uptime(0xc420158300)
        /root/work/src/github.com/virtua-network/smartos_exporter/collector/uptime.go:59 +0x11f
github.com/virtua-network/smartos_exporter/collector.(*LoadAverageExporter).Collect(0xc420158300, 0xc4205b8240)
        /root/work/src/github.com/virtua-network/smartos_exporter/collector/uptime.go:48 +0x2f
github.com/prometheus/client_golang/prometheus.(*Registry).Gather.func2(0xc42018c050, 0xc4205b8240, 0x902200, 0xc420158300)
        /root/work/src/github.com/prometheus/client_golang/prometheus/registry.go:383 +0x61
created by github.com/prometheus/client_golang/prometheus.(*Registry).Gather
        /root/work/src/github.com/prometheus/client_golang/prometheus/registry.go:381 +0x2e1
```

--- a/collector/uptime.go
+++ b/collector/uptime.go
@@ -64,7 +64,7 @@ func (e *LoadAverageExporter) uptime() {
 
 func (e *LoadAverageExporter) parseUptimeOutput(out string) (error) {
     // we will use regex in order to be sure to catch good numbers
-    r,_ := regexp.Compile(`load average: (\d.\d+), (\d.\d+), (\d.\d+)`)
+    r,_ := regexp.Compile(`load average: (\d+.\d+), (\d+.\d+), (\d+.\d+)`)
     loads := r.FindStringSubmatch(out)
 
     load1, err := strconv.ParseFloat(loads[1], 64)

